fiwClient.ts

```diff
@@ -107,6 +107,25 @@ export const client = {
     return SingleEventSchema.parse(response.data);
   },
 
+  // fetch multiple events by their ids
+  getEventsByIds: async (ids: number[]) => {
```

Comment:
this code should be located in the component using this method. Not in the fiw http client.

Reply:
Fixed